justfile

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+_default:
+    @just --list
+
+_env:
+    #!/usr/bin/env sh
+    if [ ! -f .env ]; then
+      echo "Copying docker/config/env-dist to .env..."
+      cp docker/config/env-dist .env
+    fi
+
+# Build docker images.
+build *args='base fakesentry statsd': _env
+    docker compose --progress plain build {{args}}
+
+# Run the webapp and services.
+run *args='--attach=eliot --attach=fakesentry eliot': _env
```

Comment:
It's confusing that the target running `docker compose up` is called `run`. Most other targets are named according to the Docker Compose command they run. Moreover, there's also `docker compose run`, which does something else. I'm not sure we should actually change this, since this has always been called `run`. I mainly want to hear what you think. :)

(There's also `docker compose start`, which is similar to `docker compose up`, except less useful in completely unclear ways. I like that we don't have a just target for that.)

Reply:
I think it's a great idea to rename these just commands to `just up` and `just run` to match `docker compose`

Reply:
per discussion in zoom, only `just run` should be moved to `just up`, and we'll keep `just shell`